src/test/calibrationlib/calibration_lqp_generator_test.cpp

```diff
@@ -0,0 +1,67 @@
+#include "base_test.hpp"
+
+#include <operators/get_table.hpp>
+#include <synthetic_table_generator.hpp>
+#include "storage/table.hpp"
+
+#include "calibration_lqp_generator.hpp"
+#include "calibration_table_generator.hpp"
+
+namespace opossum {
+class CalibrationLQPGeneratorTest : public BaseTest {
+ protected:
+  void SetUp() override {
```

Comment:
Looks like all tables are created for each and every test. How long does that test suite run?